esp/src/eclwatch/nls/hu/hpcc.js

```diff
@@ -605,6 +610,7 @@ define(
     ZeroLogicalFilesCheckFilter: "Nincs a megadott feltételnek megfelelő adat! (A jó a szűrőfeltétel?)",
     Zip: "Zip",
     ZippedAnalysisPackage: "Tömörített elemzési csomag",
+    Zoom: "Nagyítás",
     Zoom100Pct: "Nagyitás 100%-ra",
```

Comment:
It seems I made a mistake in some previous translation. This should be "Nagyítás 100%-ra"